.github/workflows/ci.yml

```diff
@@ -56,26 +56,8 @@ jobs:
       - run: |
           rustup set auto-self-update disable
           rustup toolchain install ${RUST_VERSION} --profile minimal
-      - name: Release Disk Quota
```

Comment:
The disk may be not enough to run the ci, so how about keeping this step?

Reply:
Solved